stl/inc/ranges

```diff
@@ -1435,20 +1436,101 @@ namespace ranges {
     template <class _Rng>
     inline constexpr bool enable_borrowed_range<ref_view<_Rng>> = true;
 
+    // clang-format off
+    template <range _Rng>
+        requires (movable<_Rng> && !_Is_specialization_v<remove_cvref_t<_Rng>, initializer_list>)

```

Comment:
I would prefer if we could put that into a named concept

Reply:
On first read I misinterpreted this as a request to put `_Is_specialization_v<remove_cvref_t<_Rng>, initializer_list>` into an `_Is_initializer_list` helper so this would more closely reflect the wording from the working draft. This makes a ton of sense to me.

Your comment on 1527, however, implies that you want the entire *requires-clause* (and presumably the *type-constraint* as well) pulled out into a helper concept. I'm inclined not to do so, again because I'd like the wording to more closely reflect the working draft. Do you have a motivating concern?